packages/shopify-app-session-storage-mysql/src/mysql.ts

```diff
@@ -153,6 +153,10 @@ export class MySQLSessionStorage implements SessionStorage {
       `;
       await this.query(query);
     }
+
+    // Update for existing tables
+    await this.query(`ALTER TABLE ${this.options.sessionTableName} 
```

Comment:
I wonder if we can do this in a way that doesn't require us to run this code every time we start up, but I can't think of a clean way other than setting up migrations like we did for sqlite.

Reply:
As discussed I'll generalized migration for all session storage and allow it to be customizable as well.